benchmark/reconstruction/evaluation/utils.py

```diff
@@ -753,8 +777,8 @@ def compute_avg_auc(scene_metrics: dict[str, SceneMetrics]) -> list[float]:
 
 
 def diff_metrics(
-    metrics_a: dict[str, dict[str, SceneMetrics]],
-    metrics_b: dict[str, dict[str, SceneMetrics]],
+    metrics_a: dict[str, dict[str, dict[str, SceneMetrics]]],
```

Comment:
Consider defining type aliases to increase readability, e.g.
```python
from typing import TypeAlias
MetricsByScene: TypeAlias = dict[str, SceneMetrics]
MetricsByCatByScene: TypeAlias = dict[str, MetricsByScene]
MetricsByDatasetByCatByScene: TypeAlias = dict[str, MetricsByCatByScene]
```
And maybe rename `SceneMetrics` to `Metrics`?